src/service/vaults.ts

```diff
@@ -110,6 +112,8 @@ type VaultFactoryParamsUpdate = ValuePossessor<{
 
 type AgoricInstancesUpdate = ValuePossessor<Array<[string, unknown]>>;
 
+type VaultUpdate = ValuePossessor<VaultInfoChainData>;
+
 export const watchVaultFactory = (netconfigUrl: string) => {
   let isStopped = false;
```

Comment:
looks like nothing changes this yet? if so please leave a comment explaining why it should stay here.

Reply:
`watchVaultFactory` returns a closure which sets `isStopped` to `true` and it's invoked when the vaults view is unmounted https://github.com/Agoric/dapp-inter/blob/main/src/views/Vaults.tsx#L14-L18. Currently we only have the vaults view but when we eventually have psm, liquidations, bld boost etc. that would happen when the user navigates away.

Reply:
ah right, I didn't see that.